### PR TITLE
Feature/add instrument wiserep

### PR DESCRIPTION
**Pull request by abostroem**
Add the option to pass an instruments dictionary to the wise rep upload script so you don't have to enter known instruments

### CODE REVIEW

File: lightcurve_fitting/speccal.py [hunk 1]
```diff
@@ -435,7 +435,7 @@ def calibrate_spectra(spectra, lc, filters=None, order=0, subtract_percentile=No
         return fig
 
 
-def create_wiserep_tsv(specpaths, wiserep_dir, verbose=False):
+def create_wiserep_tsv(specpaths, wiserep_dir, verbose=False, instruments=None):
```
Comment (griffin-h) [inline lightcurve_fitting/speccal.py:439]: Can you add `instruments` to the docstring?

File: lightcurve_fitting/speccal.py [hunk 2]
```diff
@@ -464,7 +464,8 @@ def create_wiserep_tsv(specpaths, wiserep_dir, verbose=False):
     # collect the metadata
     bibcode = input('bibcode: ')
     rows = []
-    instruments = {}
+    if not instruments:
```
Comment (griffin-h) [inline lightcurve_fitting/speccal.py]: `if instruments is None` would be a tiny bit safer I guess

**Review by griffin-h — CHANGES_REQUESTED** [review]

**Comment by griffin-h** [comment]
@abostroem could you also add this change to the [release notes](https://github.com/griffin-h/lightcurve_fitting/blob/dev/docs/source/release-history.rst)?

**Comment by abostroem** [comment]
yes, and I'll update the doc strings - but I think I found a bug - let me sort that out

File: lightcurve_fitting/speccal.py (hunk 2 above)
Comment (abostroem) [reply lightcurve_fitting/speccal.py]: fixed!

File: lightcurve_fitting/speccal.py (hunk 1 above)
Comment (abostroem) [reply lightcurve_fitting/speccal.py:439]: done!

**Review by griffin-h — APPROVED** [review]

**Pull request merged**